src/libraries/Directory.Build.props

```diff
@@ -293,8 +293,8 @@
   <PropertyGroup>
     <EnableDefaultItems>false</EnableDefaultItems>
 
-    <!-- Clear the init locals flag on all src projects, except those in VB, where we can't use spans. -->
-    <ILLinkClearInitLocals Condition="'$(IsSourceProject)' == 'true' and '$(Language)' != 'VB'">true</ILLinkClearInitLocals>
+    <!-- Clear the init locals flag on all src C# projects that have .NET Core as a TFM. Exclude VB (we can't use spans), F# (no project written in F#), and IL. -->
```

Comment:
> we can't use spans

SkipLocalsInit helps even with code that does not use Spans, so this not quite accurate.

I think this whole comment can be deleted.